test/AllReduceTests.cpp

```diff
@@ -102,6 +102,30 @@ namespace RcclUnitTesting
     testBed.Finalize();
   }
 
+  TEST(AllReduce, Channels)
+  {
+    TestBed testBed;
+    if(testBed.ev.isGfx94) {
+      // Configuration
+      std::vector<ncclFunc_t>     const funcTypes       = {ncclCollAllReduce};
+      std::vector<ncclDataType_t> const dataTypes       = {ncclUint8};
```

Comment:
please use half and bfloat16, which are most likely being used in real world apps